lib/app.dart

```diff
@@ -14,6 +16,15 @@ class MyApp extends StatefulWidget {
 }
 
 class MyAppState extends State<MyApp> {
+  @override
+  void initState() {
+    super.initState();
+    NotificationService().requestPermission();
```

Comment:
This can be initialised with ` getIt<NotificationService>().requestPermissions();`